src/controllers/classesController.js

```diff
@@ -294,6 +303,23 @@ const deleteClass = async (req, res) => {
 
 //apply for class
 
+// Function to calculate age from date of birth
+const calculateAge = (dateOfBirth) => {
```

Comment:
we have a very same function in utils/adultValidation. you can use it here

Reply:
Done